examples/federated-learning/main.py

```diff
@@ -85,18 +91,24 @@ class ModelOwner(BaseModelOwner):
                  representing the model owner.
   """
   @classmethod
```

Comment:
why are these class methods btw?

Reply:
@jvmancuso did this part!

Reply:
I wrote them as classmethods in the base class, but they don't need to be classmethods in the subclass. in this case, the reason is that the model_fn is meant to be executed on the DataOwner and thus shouldn't depend critically on any state from the ModelOwner